### DISCUSSION TITLE
Bringing Scratch 1.4 Stage menu back (at least some options)

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#7060] [Ideas]
Scratch 1.4 had a handy Stage menu that allowed users (especially very young ones) to:

1. select a rectangular area (by dragging the mouse) to create a new sprite/costume
2. save the whole stage as an image to export
3. resize the sprite
4. rotate the sprite

Option 1 was really useful to create new sprites that identified areas where other sprites could go to (by identifying Stage positions without having to use cartesian coordinates).
Option 2 was really useful to easily create slideshows by capturing the Stage during project execution
Option 3 was very nice as it showed a handle close to the sprite on the Stage to resize it. It could be more easily redesigned by highlighting the "size" box in the information area and moving the cursor to that area
Option 4 was similar to option 3 (it showed a handle to rotate the sprite on the Stage). It could be more easily also redesigned by making the direction dial show up in the information area (and maybe moving the cursor to that area)

Option 3 and 4 very nice but not essential. Instead, option 1 and 2 would be very useful when working with young students.

**Comment by Samq64** [comment]
For the second one see #3715. I don't think the others have issues yet.

**Reply by sfederici** [reply]
Yes, it is the same proposal. But I suggest adding the option to a contextual menu added to the Stage

**Comment by DNin01** [comment]
Adding a custom context menu to the stage would prevent the user from accessing the browser right-click context menu for the stage, so to avoid that problem, we could show the browser's default right-click context menu after a second right-click (which is what YouTube does, BTW).
